src/html5/parser/quirks.rs

```diff
@@ -156,15 +156,13 @@ static LIMITED_QUIRKS_PUB_IDENTIFIER_PREFIX_NOT_MISSING_SYS: &[&str] = &[
 #[cfg(test)]
 mod tests {
     use crate::html5::input_stream::InputStream;
-    use crate::html5::parser::document::DocumentBuilder;
     use crate::html5::parser::Html5Parser;
     use crate::html5::parser::QuirksMode;
 
     #[test]
     fn test_quirks_mode() {
-        let doc = DocumentBuilder::new_document();
-        let mut stream = InputStream::new();
-        let parser = Html5Parser::new(&mut stream, doc.clone());
+        let stream = &mut InputStream::new();
+        let parser = Html5Parser::new_parser(stream);
```

Comment:
Since it's returning a parser, how about just `::new(stream)`, or `::from_stream(stream)`?

Reply:
i think this makes sense  (actually, go formatting will complain about stuttering in package  / module names).

However, i want to keep this for now away from this PR, and do a next run when cleaning up things like this